frontend/app/globals.css

```
@@ -25,11 +25,23 @@
   }
 }
 
-::selection {
+*:not(.ProseMirror):not(.ProseMirror *)::selection {
```

Stylelint (17.6.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)